webpack.config.js

```diff
@@ -84,7 +84,23 @@ module.exports = (env, argv) => {
                         {
                             loader: 'style-loader',
                             options: {
-                                insertAt: 'top'
+                                insert: function insertAtTop(element) {
+                                    var parent = document.querySelector('head');
+                                    // eslint-disable-next-line no-underscore-dangle
+                                    var lastInsertedElement =
+                                        window._lastElementInsertedByStyleLoader;
+
+                                    if (!lastInsertedElement) {
+                                        parent.insertBefore(element, parent.firstChild);
```

Comment:
can we run into a possibility that parent is null? if yes then this would crash.

Reply:
Oh interesting, good eye!

All browsers on my machine add a `<head>` automatically if your page doesn't explicitly include one - for example if I load this:
```html
<!DOCTYPE html>
<html>
<body>
<h1>Hi!</h1>
</body>
</html>
```

So I think we're OK, but if this does lead to a bug report at any point we'll have an idea where to look.